apps/create-fuels-counter-guide/src/lib.ts

```diff
@@ -1,11 +1,21 @@
 import { BN, WalletLocked, WalletUnlocked } from 'fuels';
 
-export const NODE_URL = `http://127.0.0.1:${
-  process.env.NEXT_PUBLIC_FUEL_NODE_PORT || 4000
-}/graphql`;
+// #region deploying-dapp-to-testnet-lib-current-environment
+type DappEnvironment = 'local' | 'testnet';
+
+export const CURRENT_ENVIRONMENT: DappEnvironment =
+  (process.env.NEXT_PUBLIC_DAPP_ENVIRONMENT as DappEnvironment) || 'local';
+// #endregion deploying-dapp-to-testnet-lib-current-environment
+
+export const NODE_URL =
```

Comment:
You have specified this variable [here](https://github.com/FuelLabs/fuels-ts/pull/2052/files#diff-1734eb3071d7248ecb719fb50410a28fb73846295ac884e4d0fb736df4d52ad3R13) as well. Do we need both?